### DISCUSSION TITLE
how to I keep the wiki.js starting up?

### DISCUSSION THREAD

**Discussion by fandd1234** [requarks/wiki#4616] [Help / Questions] [answered]
Whenever I quit my putty session, the wiki.js will be down until I execute "node server" again.
May I ask what can I do avoiding this please?  Thanks. :)

**Comment by ieaster1** [comment, marked as answer]
You will need to run the application as a service in order for it to persist.  There are several ways to do it but recommend you start with this: https://docs.requarks.io/install/linux#run-as-service

**Reply by fandd1234** [reply]
Thank you and I am able to use forever to start its service now. :)

**Reply by ieaster1** [reply]
Glad you got everything running @fandd1234!  Please feel free to mark my reply as the answer.

See: https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer
